tests/pdctl/scheduler/scheduler_test.go

```diff
@@ -364,6 +364,30 @@ func (s *schedulerTestSuite) TestScheduler(c *C) {
 	mustExec([]string{"-u", pdAddr, "scheduler", "remove", "balance-hot-region-scheduler"}, nil)
 	mustExec([]string{"-u", pdAddr, "scheduler", "add", "balance-hot-region-scheduler"}, nil)
 	c.Assert(conf1, DeepEquals, expected1)
+
+	// test balance leader config
+	conf = make(map[string]interface{})
+	conf1 = make(map[string]interface{})
+	expectedLeader := map[string]interface{}{
+		"batch": 5,
+	}
+	mustExec([]string{"-u", pdAddr, "scheduler", "config", "balance-leader-scheduler", "show"}, &conf)
+	c.Assert(conf["batch"], Equals, 5.)
+	mustExec([]string{"-u", pdAddr, "scheduler", "config", "balance-leader-scheduler", "set", "batch", "3"}, nil)
+	expectedLeader["batch"] = 3
+	mustExec([]string{"-u", pdAddr, "scheduler", "config", "balance-leader-scheduler"}, &conf1)
+	c.Assert(conf1["batch"], Equals, 3.)
+	echo = mustExec([]string{"-u", pdAddr, "scheduler", "add", "balance-leader-scheduler"}, nil)
+	c.Assert(strings.Contains(echo, "Success!"), IsFalse)
+	echo = mustExec([]string{"-u", pdAddr, "scheduler", "remove", "balance-leader-scheduler"}, nil)
+	c.Assert(strings.Contains(echo, "Success!"), IsTrue)
+	echo = mustExec([]string{"-u", pdAddr, "scheduler", "remove", "balance-leader-scheduler"}, nil)
+	c.Assert(strings.Contains(echo, "404"), IsTrue)
```

Comment:
using `balance-leader-scheduler not enable` is better.

Reply:
There is no way to tell whether the scheduler is removed or not.  Handler just returns "not found"